obspy/signal/spectral_estimation.py

```diff
@@ -1285,6 +1297,194 @@ def _add_npz(self, filename):
             msg = msg % (duplicates, len(_times_processed), filename)
             warnings.warn(msg)
 
+    def plot_spectrogram(self, cmap=None, clim=None, grid=True, filename=None,
```

Comment:
Why not just pass the colormap as a keyword argument instead of the `is None` magic?